src/color_pixel.hpp

```diff
@@ -50,5 +50,14 @@ class ColorPixel {
      */
     int getGreen() const;
 };
-
+/**
+ * @brief Operator== for two color pixels
+ * [BLANK]
```

Comment:
I don't think you're supposed to actually leave "[blank]" in the doxygen comments?

Reply:
(I've seen this at a few places)